resources/js/Pages/Collections/Components/CollectionsFilter/CollectionsSorting.tsx

```diff
@@ -12,6 +12,8 @@ interface Properties {
 }
 
 export const CollectionsSorting = ({ disabled = false, activeSort, onSort }: Properties): JSX.Element => {
+    const { t } = useTranslation();
```

Comment:
React manages hooks by the call order, that is why I moved this call to the top of the component to prevent possible errors.